src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -160,6 +160,11 @@
 (def subject-alt-names-oid
   "2.5.29.17")
 
+;; Extension that is checked when allowing access to the certificate_status(es)
+;; endpoint. Should only be present on the puppet master cert.
+(def cli-auth-oid
+  "1.3.6.1.4.1.34380.1.3.39")
```

Comment:
Should we be using a "crazier" number? What are valid values, and what is more likely to conflict with things people might already be using?

Reply:
I think we want to disallow adding this particular custom extension to any old cert, restricting it to the generated master's cert. But I assume we should still document it in that list, with a note about how it's reserved.

Of course if this whole approach seems weird, please let us know. We wanted to be more restrictive than just allowing "localhost" to access these endpoints.